test/e2e/windows/memory_limits.go

```diff
@@ -168,11 +125,41 @@ func newMemLimitTestPods(numPods int, imageName, podType string, memoryLimit str
 				},
 			},
 		}
-
-		pods = append(pods, &pod)
+		_, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(context.TODO(), pod, metav1.CreateOptions{})
+		framework.ExpectNoError(err)
+	}
+	failurePod := &v1.Pod{
+		Spec: v1.PodSpec{
```

Comment:
Both this and the other pod spec are missing the pod name and container name (see the original pod spec definition for reference)

I would suggest this one be called something like "mem-failure-pod" and the other pods be the "mem-test-<nodename>"